src/bundle.c

```diff
@@ -552,13 +560,29 @@ int install_bundles_frontend(char **bundles)
 
 	for (; *bundles; ++bundles) {
 		bundles_list = list_prepend_data(bundles_list, *bundles);
+		if (*bundles) {
+			if (bundles_list_str) {
+				string_or_die(&bundles_list_str, "%s, %s", bundles_list_str, *bundles);
```

Comment:
Or rather, store the old `bundles_list_str` in a temp variable before freeing, and use the temp variable in the `string_or_die()`.